### DISCUSSION TITLE
Sujet / Thema 11 –– Geodaten in verschiedenen Bezugssystemen / Données géographiques dans différents systèmes de référence

### DISCUSSION THREAD

**Discussion by rsalzer** [GeoUnconference/discussions#27] [Ideen / Idées]
Einige Geoprodukte (glücklicherweise nicht alle) werden einzig im CH-Format LV95 (EPSG:2056) angeboten. Dies macht bei vielen Tools eine Umprojektion nötig. 
Es wäre wünschenswert, so viele Produkte wie möglich in mehreren Projektionen anzubieten, vorzugsweise in den gebräuchlichsten (bspw. WGS84, EPSG:4326), so dass Einstiegshürden abgebaut werden.

**Comment by davidoesch** [comment]
die Geodienste des Bundes WMS  https://www.geo.admin.ch/de/geo-dienstleistungen/geodienste/darstellungsdienste-webmapping-webgis-anwendungen/web-map-services-wms.html WMTS https://www.geo.admin.ch/de/geo-dienstleistungen/geodienste/darstellungsdienste-webmapping-webgis-anwendungen/web-map-tiling-services-wmts.html  und auch das RESTful API https://api3.geo.admin.ch/services/sdiservices.html unterstützen den WGS84 und webmercator

Im donwload sind jedoch meist die Anlieferprojektion vorhanden. Einige  sind jedoch auch in WGS84 vorhanden

**Reply by rsalzer** [reply]
Genau: Viel Wildwuchs. Mir fiel es beispielsweise bei den Gemeinde- oder Bezirksgrenzen auf (hab ich mal für eine Covid-Visualisierung gebraucht). Da musste ich selbst umprojezieren...
